core/jobs/batch_jobs/suggestion_stats_computation_jobs_test.py

```diff
@@ -367,10 +356,12 @@ def test_creates_stats_model_from_one_in_review_suggestion_with_opportunity(
             translation_stats_model.rejected_translation_word_count, 0)
         self.assertItemsEqual(
             translation_stats_model.contribution_dates,
-            [datetime.date.today()]
+            [datetime.datetime.utcnow().date()]
         )
 
-    def test_creates_stats_model_from_one_accepted_suggestion(self) -> None:
+    def test_creates_translation_stats_model_from_one_accepted_suggestion(
```

Comment:
```suggestion
    def test_creates_translation_stats_models_from_one_accepted_suggestion(
```

To make it clear that multiple stats models should be created.

Reply:
Done